xml/System.Transactions/Enlistment.xml

```diff
@@ -50,7 +50,7 @@
     <threadsafe>This type is thread safe.</threadsafe>
     <altmember cref="T:System.Transactions.PreparingEnlistment" />
     <altmember cref="T:System.Transactions.IEnlistmentNotification" />
-    <related type="Article" href="https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/ee818750(v=vs.100)">Enlisting Resources As Participants In A Transaction</related>
+    <related type="Article" href="https://docs.microsoft.com/en-us/previous-versions/ms172153(v=vs.90)">Enlisting Resources As Participants In A Transaction</related>
```

Comment:
The previous version seemed a bit better since it has TOC

```suggestion
    <related type="Article" href="~/docs/framework/data/transactions/enlisting-resources-as-participants-in-a-transaction.md">Enlisting Resources As Participants In A Transaction</related>
```